clients/drcachesim/tracer/raw2trace.h

```diff
@@ -85,25 +189,8 @@ class raw2trace_t {
     ~raw2trace_t();
 
     /**
-     * Adds handling for custom data fields that were stored with each module via
-     * drmemtrace_custom_module_data() during trace generation.  When do_conversion()
-     * or do_module_parsing() is subsequently called, its parsing of the module data
-     * will invoke \p parse_cb, which should advance the module data pointer passed
-     * in \p src and return it as its return value (or nullptr on error), returning
-     * the resulting parsed data in \p data.  The \p data pointer will later be
-     * passed to both \p process_cb, which can update the module path inside \p info
-     * (and return a non-empty string on error), and \b free_cb, which can perform
-     * cleanup.
-     *
-     * A custom callback value \p process_cb_user_data can be passed to \p
-     * process_cb.  The same is not provided for the other callbacks as they end up
-     * using the drmodtrack_add_custom_data() framework where there is no support for
-     * custom callback parameters.
```

Comment:
I suppose this is a meta-comment but I wonder if you explicitly made a decision to remove it or it just got accidentally lost?